backend/src/book_add.ts

```diff
@@ -1,33 +1,55 @@
 import config from "./mikro-orm.config";
 import { Book } from "./entities/book.entity";
+import { Genre } from "./entities/genre.entity";
+import { BookGenre } from "./entities/book_genres.entity";
 import { MikroORM } from "@mikro-orm/postgresql";
 
 async function addTestBook() {
     const orm = await MikroORM.init(config);
     const em = orm.em.fork();
-    
-    try {
-        // const newBook = em.create<Book>('Book', {
-        //     bookId: 5,
-        //     title: 'Test 1',
-        //     author: 'John Doe',
-        //     synopsis: 'Some text',
-        //     datePublished: new Date('2024-05-25'),
-        //     rating: 0.0,
-        //     coverImage: 'path/to/image.png'
-        // });
 
-        const newBook = new Book(
-            6,
-            'Test 2',
-            'John Doe',
+    try {
+        /* const newBook = new Book(
+            8,
+            'Test 5',
+            'Jane Doe',
             'Some text',
-            new Date('2024-05-25'),
+            ["Horror", "Romance"],
+            new Date('2024-05-26'),
             0.0,
             'path/to/image.png'
+        ); */
+
+        const newBook = new Book(
+            'Test 16',
+            'John Doe',
+            'A great book',
+            ["Comedy"],
+            'path/to/image.png',    
         );
 
-        await em.persist(newBook).flush();
+
+        await em.persist(newBook).flush()
+        .then(async () => {
```

Comment:
I do like the overall style here, we will probably do something similar in the book builder, we'd just need a catch case at the end.